pkg/clients/cliServiceClient.go

```diff
@@ -32,17 +34,24 @@ type CliClient struct {
 }
 
 func NewCliServiceClient(url string, networkValidator cliClient.NetworkValidator) *CliClient {
+	state := servicestate.New()
```

Comment:
same as this comment:
https://github.com/datreeio/cluster-scan-job/pull/52#discussion_r1118075730